samples/index.md

```diff
@@ -1,41 +1,20 @@
-| [Docker for Beginners]({{ labsbase }}/beginner/){: target="_blank"}                                                   | A good "Docker 101" course.                                                                                                                 |
-| [Docker Swarm mode]({{ labsbase}}/swarm-mode){: target="_blank"}                                                      | Use Docker for natively managing a cluster of Docker Engines called a swarm.                                                                |
-| [Configuring developer tools and programming languages]({{ labsbase }}/developer-tools/README.md){: target="_blank"}  | How to set-up and use common developer tools and programming languages with Docker.                                                         |
-| [Live Debugging Java with Docker]({{ labsbase }}/developer-tools/java-debugging){: target="_blank"}                   | Java developers can use Docker to build a development environment where they can run, test, and live debug code running within a container. |
-| [Docker for Java Developers]({{ labsbase }}/developer-tools/java/){: target="_blank"}                                 | Offers Java developers an intro-level and self-paced hands-on workshop with Docker.                                                         |
-| [Live Debugging a Node.js application in Docker]({{ labsbase }}/developer-tools/nodejs-debugging){: target="_blank"}  | Node developers can use Docker to build a development environment where they can run, test, and live debug code running within a container. |
-| [Dockerizing a Node.js application]({{ labsbase }}/developer-tools/nodejs/porting/){: target="_blank"}                | This tutorial starts with a simple Node.js application and details the steps needed to Dockerize it and ensure its scalability.             |
-| [Docker for ASP.NET and Windows containers]({{ labsbase }}/windows/readme.md){: target="_blank"}                      | Docker supports Windows containers, too! Learn how to run ASP.NET, SQL Server, and more in these tutorials.                                 |
-| [Docker Security]({{ labsbase }}/security/README.md){: target="_blank"}                                               | How to take advantage of Docker security features.                                                                                          |
-| [Building a 12-factor application with Docker]({{ labsbase}}/12factor){: target="_blank"}                             | Use Docker to create an app that conforms to Heroku's "12 factors for cloud-native applications."                                           |
-
-## Sample applications
-
-Run popular software using Docker.
+## Samples in this documentation
```

Comment:
Most of the samples in this list should also be removed or replaced; some of them were written 8 years ago, and never properly updated (and have been cause for confusion as users saw them as "best practice") 😔

I'm not sure if we have good content to replace them though

Reply:
They have now all been removed from the table and TOC. I won't delete the topics as of yet because a few are still used as examples in other topics.